libcodechecker/server/client_db_access_handler.py

```diff
@@ -1245,9 +1219,19 @@ def removeRunResults(self, run_ids):
             runs_to_delete.append(run_to_delete)
 
         for run_to_delete in runs_to_delete:
+            # FIXME: clean up bugpaths. Once run_id is a foreign key there,
+            # it should be automatic.
             session.delete(run_to_delete)
             session.commit()
 
+        s1 = select([BugPathEvent.file_id])
```

Comment:
Let's make this code more verbose. It's really clunky to read, especially the `~something` instead of `not something`.

But if this code doesn't do **anything** (seemingly it does not...), we shouldn't introduce it in the first place.

At this point, when I try to delete a run from the database, I get the following error in the server console:

    [ERROR] [12:05] - 'DeclarativeMeta' object is not iterable

And while run is deleted, the files and file_contents table is left intact.